Update pim docs 2.0 for the projects console (#1060)
akeneo/pim-docs: cloud_edition/serenity_mode/docs/overview.rst

Overview
--------

To start, you need to set up your serenity instances from the `Portal <https://help.akeneo.com/portal/articles/set-up-akeneo-serenity.html?utm_source=akeneo-docs&utm_campaign=serenity_overview>`_

Akeneo Cloud Serenity provides 2 environments:

+--------------------------------------+--------------------------------------+
| - The **production** environment:    | my-project.cloud.akeneo.com          |
+--------------------------------------+--------------------------------------+

> | - The **sandbox** environment:       | my-project-staging.cloud.akeneo.com  |

Author: Sorry here, but It's intentional: the naming changed, the type of the non-prod instance of the environment is now named "sandbox". I that type of environment Development/Trunk, Integration, Staging, Preprod, etc... See the Project Console documentation.

> | - The **sandbox** environment:       | my-project-staging.cloud.akeneo.com  |

Reviewer: Awesome!

+--------------------------------------+--------------------------------------+

.. warning::

>     **Warning:** Only the production environment is backuped. All data produced on the sandbox environment are not backed up.

Reviewer: backed up
```suggestion
    **Warning:** Only the production environment is backed up. Any data produced on the staging environment are not
```

>     **Warning:** Only the production environment is backuped. All data produced on the sandbox environment are not backed up.

Reviewer 2: ```suggestion
    **Warning:** Only the production environment is backed up. All data produced on the sandbox environment are not backed up.
```

> .. note::

Reviewer: seems to obvious to be stated but why not after all ;)

Author: I take as model the flexibility page.
